File: js/stockPicker.js
exports.picker = function (prices) {
  let highestRes = 0;
  let highestPair = [];
  for (let i = 0; i < prices.length; i++) {
    for (let j = i + 1; j < prices.length; j++) {
      let result = prices[j] - prices[i];
      if (result > highestRes) {
        highestRes = result;
        highestPair = [i, j];
      }
    }
  }
  return highestPair;
};

File: python/stock_picker.py
def picker(prices):
    highestRes = 0;
    highestPair = [];
    for i in range(len(prices)):
        for j in range(i + 1, len(prices)):
            result = prices[j] - prices[i];
            if result > highestRes:
                highestRes = result;
                highestPair = [i, j]
      
    return highestPair;